deepinv/utils/demo.py

```diff
@@ -78,9 +79,28 @@ def get_image_url(file_name):
     )
 
 
+def get_data_home():
+    """Return folder to store data in.
+
+    This folder can be set by setting the environment variable``DEEPINV_DATA``,
+    or ``XDG_DATA_HOME``. By default, it will be set in ``./datasets``.
+    """
+    data_home = os.environ.get("DEEPINV_DATA", None)
+    if data_home is not None:
+        return Path(data_home)
+
+    data_home = os.environ.get("XDG_DATA_HOME", None)
+    if data_home is not None:
+        return Path(data_home) / "deepinv"
+
+    return Path(".") / "datasets"
+
+
 def load_dataset(
-    dataset_name, data_dir, transform, download=True, url=None, train=True
+    dataset_name, transform, data_dir=None, download=True, url=None, train=True
```

Comment:
```suggestion
    dataset_name, transform, *, data_dir=None, download=True, url=None, train=True
```
As we are changing the order of the args, maybe safer to require that the name are specified if a user uses this ?